manifests/s/sideloadly/sideloadly/0.25.0/sideloadly.sideloadly.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: sideloadly.sideloadly
+PackageVersion: 0.25.0
+PackageLocale: en-US
+Publisher: Sideloadly
+PackageName: Sideloadly
+License: Unknown

```

If the license is not known, assume it is proprietary
```suggestion
License: Proprietary
```